static/templates/status_emoji_name_tooltip_content.hbs

```diff
@@ -0,0 +1,3 @@
+<div class="status_emoji_name_tooltip_content">
+    <span class="tooltip_inner_content">{{first_line}}</span>
```

Comment:
Why the variable name `first_line` here?

Reply:
I had thought that if multiple lines will be needed then 'first_line' would make sense.
actually, the tooltip is only for emoji names so I have changed it from 'first_line' to 'status emoji name' in my next commit.